src/ec/suite_b/ops.rs

```diff
@@ -240,7 +240,7 @@ pub struct ScalarOps {
     pub common: &'static CommonOps,
 
     scalar_inv_to_mont_impl: fn(a: &Scalar) -> Scalar<R>,
-    scalar_mul_mont: unsafe extern "C" fn(r: *mut Limb, a: *const Limb, b: *const Limb),
+    scalar_mul_mont: fn(r: &mut [Limb], a: &[Limb], b: &[Limb]),
```

Comment:
Ditto: We'll have to leave this as it was so that the P256 implementation can directly call the assembly/C code without any additional overhead.